libs/wingsdk/src/std/array.ts

```diff
@@ -0,0 +1,48 @@
+// These classes are used by Wing to provide JSII subsets of the JS Array class.
+// They should not be consumed directly by users.
+// TODO: These should be interfaces, currently Wing does not support interface JSII imports
+
+/**
+ * Immutable Array
```

Comment:
(or @wing-name)

```suggestion
 * Immutable Array
 * @name Array
```

Reply:
This is similar to the `@typeparam` thing where I'm concerned about preemptively adding too much use of JSII metadata when it's not technically necessary for this PR.

Reply:
Nit picking not important.

Reply:
No problem